assets/sass/protocol/base/elements/_links.scss

```diff
@@ -4,61 +4,50 @@
 
 @use '../../includes/lib' as *;
 
-:link {
-    color: $link-color;
+a {
+    color: var(--link-color);
     text-decoration: underline;
 
-    &:hover,
-    &:focus,
-    &:active {
-        color: $link-color-hover;
+    &:where(:hover, :focus, :active) {
+        color: var(--link-color-hover);
         text-decoration: none;
     }
 
-    &:active {
+    &:where(:active) {
         background-color: rgba(0, 0, 0, 0.05);
     }
-
-    @supports (--css: variables) {
-        color: var(--link-color);
-
-        &:hover,
-        &:focus,
-        &:active {
-            color: var(--link-color-hover);
-        }
-    }
 }
 
-:visited {
-    color: $link-color-visited;
-
-    &:hover,
-    &:focus,
-    &:active {
-        color: $link-color-visited-hover;
-    }
-
-    @supports (--css: variables) {
-        color: var(--link-color-visited);
+a:where(:visited) {
+    color: var(--link-color-visited);
 
-        &:hover,
-        &:focus,
-        &:active {
-            color: var(--link-color-visited-hover);
-        }
+    &:where(:hover, :focus, :active) {
+        color: var(--link-color-visited-hover);
     }
 }
 
+// any descendant link will inherit dark theming (replace light-links mixin)
+// this might belong in root file? site-wide dark theme class
 .mzp-t-dark {
-    @include light-links;
+    --link-color: var(--link-color-inverse);
+    --link-color-hover: var(--link-color-hover-inverse);
+    --link-color-visited: var(--link-color-visited-inverse);
+    --link-color-visited-hover:var(--link-color-visited-hover-inverse);
+}
+
+// any descendant link will inherit white theming (replace white-links mixin)
+// link-specific theme class
+.mzp-t-white-links {
```

Comment:
Inherited values
<img width="438" alt="Screenshot 2025-04-16 at 12 37 39 PM" src="https://github.com/user-attachments/assets/20353388-c430-44cf-ae47-e7c33b5b3338" />

Reply:
I love the move the variables here! We need to be careful we also move to variables for the background colours _at the same time_ so we don't get in a circumstance where the background is defined as dark using a hex colour but the component depends on variables for the links.

Perhaps the prudent thing to do is to bundle the CSS variable changes into one big point release version that we test on un-supporting browsers.